blocks/cards/cards.css

```diff
@@ -33,99 +34,86 @@
   letter-spacing: var(--letter-spacing-l)
 }
 
-.cards.block .cards-card-body p {
+.cards-item .card-body p {
   margin: 8px 0 24px;
 }
 
-.cards.block .cards-card-body h3 {
+.cards-item .card-body h3 {
   padding-top: 16px;
 }
 
-.cards.block .cards-card-body .button-container {
+.cards-item .card-body .button-container {
   margin: 16px 0 0 ;
   height: 40px;
   display: flex;
 }
 
-.cards.block .cards-card-body .button-container a {
+.cards-item .card-body .button-container a {
   padding: 10px 24px;
   border: 1px solid var(--black);
   color: var(--black);
 }
 
-.cards-container .default-content-wrapper {
-  display: flex;
-  justify-content: center;
-  align-items: center;
-  padding: 32px 0;
```

Comment:
we need this padding to have an indent between the title and the card block
<img width="1221" alt="Screen Shot 2023-04-27 at 9 57 37 AM" src="https://user-images.githubusercontent.com/7125318/234903167-2cdc4648-0f38-4b90-95ec-e7547a0657b2.png">

Reply:
You can't apply any styles to the default content wrapper within the context of a Block's CSS. The default content wrapper is not associated with the block, it's the non-block content within the same section.